src/test/java/APITest/serialisation1.java

```diff
@@ -0,0 +1,42 @@
+package APITest;
+import static io.restassured.RestAssured.given;
+import ResponsePojo.response_create_user;
+import ResponsePojo.response_update_job;
+import pojo.CreatePost;
+
+public class Serialisation1 {
```

Comment:
Method/class names should not contain 1/2 etc numbers. That usually is an indication that they haven't been named properly.